frontend/packages/dev-console/integration-tests/support/pages/app.ts

```diff
@@ -199,6 +199,16 @@ export const projectNameSpace = {
     });
   },
 
+  selectProjectOrDoNothing: (projectName: string) => {
+    projectNameSpace.clickProjectDropdown();
+    cy.byLegacyTestID('dropdown-text-filter').type(projectName);
+    cy.get('[role="listbox"]').then(($el) => {
+      if ($el.find('li[role="option"]').length !== 0) {
+        cy.get(`[id="${projectName}-link"]`).click();
+      }
+    });
+  },
+
```

Comment:
Is there any reason, why we have created above function, instead of re-using [SelectOrCreateProject](https://github.com/openshift/console/blob/master/frontend/packages/dev-console/integration-tests/support/pages/app.ts#L164) function

Reply:
SelectOrCreateProject tries to create project if it doesn't exist. I needed a method, that selects the project if it exists, or otherwise does nothing.
The reason for this is that CRW will be installed into default namespace `openshift-workspaces`, but namespaces starting with `openshift-*` cannot be created by users directly. That's why, when I used `SelectOrCreateProject` the test failed (unable to create `openshift-workspaces` namespace).